config/locales/mailers.fr.yml

```
@@ -193,6 +200,21 @@ fr:
+        litige_agricole_html: 'Litige lié au domaine agricole : <ul><li>Médiateur des relations commerciales agricoles : <a href=''https://agriculture.gouv.fr/le-mediateur-des-relations-commerciales-agricoles''>https://agriculture.gouv.fr</a></li></ul>'
+        litige_assurance_html: 'Litige lié aux assurances : <ul><li>Médiateur des assurances : <a href=''https://www.mediation-assurance.org''>https://www.mediation-assurance.org</a></li></ul>'
+        litige_cotisation_html: 'Litige avec un gestionnaire des cotisations sociales : <ul><li>Médiateur de la MSA : <a href=''https://www.msa.fr/lfp/le-mediateur-de-la-msa''>https://www.msa.fr</a></li><li>Médiateur de l’URSSAF : <a href=''https://www.urssaf.fr/portail/home/utile-et-pratique/mediation.html''>https://www.urssaf.fr</a></li></ul>'
+        litige_energie_html: 'Litige lié à l’énergie : <ul><li>Médiateur National de l''énergie : <a href=''https://www.energie-mediateur.fr''>https://www.energie-mediateur.fr</a></li></ul>'
+        litige_formation_pro_html: 'Litige lié à la formation professionnelle : <ul><li>Médiateur de France Compétences : <a href=''https://www.francecompetences.fr/espace-mediation-de-france-competences''>https://www.francecompetences.fr</a></li></ul>'
+        litige_france_travail_html: 'Litige avec France Travail : <ul><li>Médiateur France Travail : <a href=''https://www.francetravail.fr/candidat/vos-droits-et-demarches/reclamations/le-mediateur-de-pole-emploi.html''>https://www.francetravail.fr</a></li></ul>'
+        litige_impots_html: 'Litige avec les impôts et services douaniers : <ul><li>Médiateur de Bercy : <a href=''https://www.economie.gouv.fr/mediateur/mediateur-bercy''>https://www.economie.gouv.fr</a></li></ul>'
+        litige_paris_idf_html: 'Litige avec la ville de Paris ou de la Région Île de France : <ul><li>Médiateur de la ville de Paris : <a href=''https://www.paris.fr/pages/le-mediateur-de-paris-2687/''>https://www.paris.fr</a></li><li>Médiateur de le région Île de France : <a href=''https://www.iledefrance.fr/saisir-le-mediateur-de-la-region-ile-de-france''>https://www.iledefrance.fr</a></li></ul>'
+        mediateurs_list: 'Voici également la liste des différents médiateurs en fonction de votre besoin :'
+        no_expert: Malheureusement, nous n’avons pas encore d’expert en mesure de répondre à votre problématique dans votre secteur d’activité.
+        recouvrement_html: 'procéder à un recouvrement amiable : par voie d’huissier, par exemple. La procédure est simplifiée jusqu’à 5 000 euros. Elle est enclenchée à l’initiative du créancier qui peut le faire directement via la plateforme de traitement des petites créances de la Chambre nationale des commissaires de justice : <a href=''https://www.credicys.fr''>https://www.credicys.fr</a>'
```

J'ai eu un traumatisme de deux points qui passent à la ligne seules étant petit
```suggestion
        recouvrement_html: 'procéder à un recouvrement amiable : par voie d’huissier, par exemple. La procédure est simplifiée jusqu’à 5 000 euros. Elle est enclenchée à l’initiative du créancier qui peut le faire directement via la plateforme de traitement des petites créances de la Chambre nationale des commissaires de justice : <a href=''https://www.credicys.fr''>https://www.credicys.fr</a>'
```